core/src/main/java/tachyon/worker/WorkerStorage.java

```diff
@@ -245,8 +245,10 @@ public void run() {
             LOG.error("Failed to rename from " + midPath + " to " + dstPath);
           }
           mMasterClient.addCheckpoint(mWorkerId, fileId, fileSizeByte, dstPath);
-          long shouldTakeMs = (long) (1000.0 * fileSizeByte / Constants.MB
-              / WorkerConf.get().WORKER_PER_THREAD_CHECKPOINT_CAP_MB_SEC);
+          long shouldTakeMs =
```

Comment:
Maven Checkstyle failed the build for me though maybe only because applying the Tachyon formatter put that line onto a single line which may be the formatter not being quite right